- https://bugs.gentoo.org/946057
- https://github.com/obsproject/obs-studio/issues/11667

From 428142d7022989af61a684a680e2d6ecc5b7ca8a Mon Sep 17 00:00:00 2001
From: Kurt Kartaltepe <kkartaltepe@gmail.com>
Date: Tue, 24 Dec 2024 10:22:40 -0800
Subject: [PATCH] libobs: Fix underlinking by adding libx11

We directly depend on libx11 for various operations in obs-nix-x11.c,
not just xcb. So be sure to link it in too.

fixes #11667
---
 libobs/cmake/os-linux.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libobs/cmake/os-linux.cmake b/libobs/cmake/os-linux.cmake
index 317e75cab0c1dd..1af874f4f1fefe 100644
--- a/libobs/cmake/os-linux.cmake
+++ b/libobs/cmake/os-linux.cmake
@@ -47,6 +47,7 @@ endif()
 target_link_libraries(
   libobs
   PRIVATE
+    X11::X11
     X11::x11-xcb
     xcb::xcb
     LibUUID::LibUUID
